Add FAQ entry for TUnit test discovery issue in Test Explorer (#8311)
microsoft/vscode-docs: docs/csharp/cs-dev-kit-faq.md

Build your solution by right-clicking on the solution in the Solution Explorer and select **Build** or `kb(workbench.action.tasks.build)`. Once the build has been completed, your tests will appear in the Test Explorer panel.

> ### Why are my TUnit tests not being discovered or run in the C# Dev Kit Test Explorer?

Reviewer: ```suggestion
### Why are my tests not being discovered or run in the Test Explorer?
```

> If you’re using the TUnit test framework, your tests might not be discovered or run in the C# Dev Kit Test Explorer because TUnit is built on the Microsoft Testing Platform (MTP), which differs from the traditional VSTest framework. To resolve this, you need to enable the "Use Testing Platform Protocol" setting in Visual Studio Code to allow the C# Dev Kit to communicate with TUnit tests.

Reviewer: ```suggestion
If your test project is using the Microsoft Testing Platform (MTP), either via MSTest, NUnit, xUnit v3, or the TUnit test framework, your tests might not be discovered or run in the Test Explorer because MTP differs from the traditional VSTest platform. To resolve this, you need to enable the "Use Testing Platform Protocol" setting in Visual Studio Code to allow the C# Dev Kit to communicate with MTP test projects.
```

Follow these steps to enable the setting:
- Open Settings in VS Code: Go to File > Preferences > Settings (or press Ctrl + ,).
- In the Settings search bar, type "Test Window" to filter results.
- Find the setting Dotnet > Test Window: Use Testing Platform Protocol under the C# Dev Kit extension settings.
- Check the box to enable it (or toggle it to "On").

> - Reload VS Code to apply the changes: Press Ctrl + Shift + P, type Developer: Reload Window, and run the command.

Reviewer 2: ```suggestion
- Reload VS Code by running the **Reload Window** command in the Command Palette (`kb(workbench.action.showCommands)`).
```

> After enabling this setting, your TUnit tests should be discovered and run properly in the Test Explorer.

Reviewer: ```suggestion
After enabling this setting, your tests should be discovered and run properly in the Test Explorer.
```
